shared-test/src/main/java/uk/gov/di/authentication/sharedtest/basetest/HandlerIntegrationTest.java

```diff
@@ -196,6 +202,26 @@ public String getEmailQueueUri() {
             return notificationQueue.getQueueUrl();
         }
 
+        @Override
```

Comment:
I'm unsure why we need both these overrides and the above changes to the `ParameterStoreExtension`?

Reply:
Thanks. We worked on this together and have refactored the code so that it no longer requires the overrides.